conductor_api/src/config.rs

```diff
@@ -222,6 +235,129 @@ impl Configuration {
         Ok(())
     }
 
+    fn check_bridge_requirements(
+        &self,
+        bridge_config: &Bridge,
+        mut dna_loader: &mut DnaLoader,
+    ) -> Result<(), String> {
+        //
+        // Get caller's config. DNA config, and DNA:
+        //
+        let caller_config = self
+            .instance_by_id(&bridge_config.caller_id)
+            .ok_or(format!(
+                "Instance configuration \"{}\" not found, mentioned in bridge",
+                bridge_config.caller_id
+            ))?;
+
+        let caller_dna_config = self.dna_by_id(&caller_config.dna).ok_or_else(|| {
+            format!(
+                "DNA configuration \"{}\" not found, mentioned in instance \"{}\"",
+                caller_config.dna, caller_config.id
+            )
+        })?;
+
+        let caller_dna_file = caller_dna_config.file.clone();
+        let caller_dna =
+            Arc::get_mut(&mut dna_loader).unwrap()(&PathBuf::from(caller_dna_file.clone()))
+                .map_err(|_| format!("Could not load DNA file \"{}\"", caller_dna_file))?;
```

Comment:
Can we have the actual error message here?  That's often helpful, i.e. perms, file not found, etc:

```suggestion
                .map_err(|err| format!("Could not load DNA file \"{}\"; error was: {}", caller_dna_file, err))?;
```